source/localizable/tutorial/subroutes.md

```diff
@@ -131,15 +131,13 @@ Now that we are returning all of our rentals to the nested route's model, we wil
 ```
 
 Finally, we need to make our controller that has our filter action available to the new nested index route.
-Instead of moving the whole controller file over to `app/controller/rentals/index.js` from `app/controller/rentals.js`, we'll just take advantage of the route's `controllerName` property to just point at our existing `rentals` controller.
+Start by running `ember g route rentals/index` to create an index controller for our nested route.
```

Comment:
@ebryn @acorncom for this section I'm using module re-export instead of `controllerName`.  Does this look ok?